### DISCUSSION TITLE
How to override automatically reset settings?

### DISCUSSION THREAD

**Discussion by saschabrockel** [RandomNinjaAtk/docker-radarr-extended#42] [Q&A] [answered]
Hey, I like the work but I'm unhappy that I configured the Any profile like I think it should be for me with x265 and stuff, and without even recognizing it, it was gone / reset. The same applies to the quality definitions. Because I guess the solution for the profile is to just create your own one. But quality definitions are reset as well every night (at least for me).

Is there a possibility to disable this? Or does it happen while the image update? Because there was one last night but the strange thing it also reset days before when no update was done.

**Comment by RandomNinjaAtk** [comment, marked as answer]
Sounds like you have recyclarr enabled, and it is updating based on the default configuration that you haven't customized. (https://github.com/RandomNinjaAtk/docker-radarr-extended/wiki/Recyclarr.bash)

Disable recyclarr and see if that resolves your issue. Also you should consider modifying the recyclarr config to match your preferences.

Edit:
Example of custom config: https://github.com/RandomNinjaAtk/docker-radarr-extended/blob/main/root/recyclarr-custom.yaml
